phpunit.xml.dist

```diff
@@ -28,6 +28,6 @@
     </filter>
 
     <php>
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0&amp;max[direct]=27"></env>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=17&amp;max[direct]=27"></env>
```

Comment:
I'm reverting this one as it should not be needed (deprecations should be fixed instead).